whisk/action.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 type ActionService struct {
-    client *Client
+    client ClientInterface
```

Comment:
should all command "services" (i.e. PackageService, etc) follow suit to maintain service consistency?

Reply:
They should be, but can be done when unit tests are added to those services.